apps/sportsstandings/sportsstandings.go

```diff
@@ -0,0 +1,25 @@
+// Package sportsstandings provides details for the Sports Standings applet.
+package sportsstandings
+
+import (
+	_ "embed"
+
+	"tidbyt.dev/community/apps/manifest"
+)
+
+//go:embed sports_standings.star
+var source []byte
+
+// New creates a new instance of the Sports Standings applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "sports-standings",
+		Name:        "Sports Standings",
+		Author:      "rs7q5 (RIS)",
+		Summary:     "Get sports standings",
+		Desc:        "Get various sports standings (currently only MLB and NHL).",
```

Comment:
Just updating the description to match the other update.
```suggestion
		Desc:        "Get various sports standings (data courtesy of ESPN).",
```